### DISCUSSION TITLE
Whitelisted domain, but error in app says it cannot validate redirect

### DISCUSSION THREAD

**Discussion by kgilliam125** [Web3Auth/Web3Auth#539] [Help]
Hi there, I am having an issue whitelisting the domain for my site. I can confirm that it is whitelisted in the web3auth dashboard and that the url matches the domain I am using. From everything I can see the setup seems correct.

Here is my setup in web3auth dashboard:
![image](https://user-images.githubusercontent.com/9299965/188334578-67d9e5cd-91a9-4462-bc87-e90ab0ae6899.png)

This is the error in the console when using web3auth login from my site:
> Error: could not validate redirect, please whitelist your domain: https://songstache.xyz for provided clientId vkqDxwao7kJf5eVfMQY1j1cOpdjsTRHn4e2LsSUb9Q6FOstcGgGLjGgbgTwJ7_CERNZ8ta0c4u05_HB6tQ at https://dashboard.web3auth.io

I have verified the Client Id in my code matches the above Client ID from the logs. I have a prototype deployed at https://songstache.xyz/alpha/explore to show the current issue.

Is anyone able to help clear this up?

**Comment by himanshuchawla009** [comment]
Hey, you are passing incorrect clientId, your correct client_id is : `BK6N-vkqDxwao7kJf5eVfMQY1j1cOpdjsTRHn4e2LsSUb9Q6FOstcGgGLjGgbgTwJ7_CERNZ8ta0c4u05_HB6tQ`

Note you are missing initial `BK6N-` prefix

**Reply by kgilliam125** [reply]
Wow, good catch. Can't believe I missed that and not sure how it worked the first time 🤦. Thanks!

**Comment by salviega** [comment]
i! HI have this page deployed but when I want to sign in don't have permissions

![image](https://user-images.githubusercontent.com/90350943/201789944-d75e3e54-91e7-4d12-96d3-916f5c812952.png)

I have my website in whitelist and it doesn't work either

![image](https://user-images.githubusercontent.com/90350943/201789993-0500b8d6-78e2-4681-83de-badf5c10e7bc.png)

**Reply by lucassilver07** [reply]
I am getting exactly same issue, it works in my local but it does not work with production domains.

**Reply by dorukozerr** [reply]
I'm having the same problem. There was an old codebase with web3auth as a wagmi connector and the URL was whitelisted. It was all working fine. Then I updated the code with the web3auth SDK, deployed the app to a test URL, and whitelisted it. However, it didn't work. I figured it might take a while to whitelist the URL. I've finally deployed it to production, but it says this domain is not whitelisted. But it was whitelisted and working.

**Comment by girishvi** [comment]
@himanshuchawla009 Can you please check?

**Comment by amberfi** [comment]
Hi, I'm using subdomains and getting the same error in testing.
This all worked earlier today, but suddenly subdomains stopped working with your infrastructure this afternoon.
I can successfully login to an app with localhost but app.localhost no longer works.

Did you guys change how you validate domains today?  Can we get this fixed asap if it's broken?
Doing a login with http://app.localhost:3000 that's giving me 
Error: could not validate redirect, please whitelist your domain: http://app.localhost:3000/ for provided clientId BFHgaoa-H_IEFjVx8w63gy5OEfGUCTTxoPkor9RmUbTscH_GTl0ui4MDlXJU1aIFRLmtHmvb2ckpzt1g0CU3tUw at https://dashboard.web3auth.io/
This app was working previously, but just stopped.  Any ideas?

**Reply by dimitrymd** [reply]
Have the same issue, precisely. We need a fix.

**Comment by dimitrymd** [comment]
Hi! I'm also using subdomains and get this kind of error. Can it be fixed somehow? Looks like it's on your end - the issue when using subdomain. 

My subdomain is whitelisted and the clientID is also correct. 

loglevelSentry.esm.js:72 Error: could not validate redirect, please whitelist your domain: https://app.bank11.io for provided clientId BI06CntgSiQ47egM5qMqkOOCzC5xN5ZeaRrHXGpb8PhJdTN1gwqX0r9okLKPcMKx5pXeJK-xl0WAb5pHpmlK4gY at https://dashboard.web3auth.io

**Reply by dimitrymd** [reply]
@himanshuchawla009 we need your help.

**Comment by billtlee** [comment]
We are also having the same error
![screenshot-2022-11-22-at-10160_112hy32](https://user-images.githubusercontent.com/14323281/203203290-1b35d70c-411d-4599-ad7e-48604d268016.png)
![screenshot-2022-11-22-at-10164_jvhank](https://user-images.githubusercontent.com/14323281/203203323-39558c27-3a44-4a91-b6de-92a1d5918e9f.png)

**Comment by billtlee** [comment]
seems this has been days without a fix.  luckily it's only happening on our testnet.  if it was production, our service would be down for 4 days.  what is the SLA at web3auth?

**Comment by AkioAlex0817** [comment]
Hello, When is possible to fix this issue?

**Comment by Haripandey21** [comment]
hey anyone fixed that above problem ?? 

i'm also getting the same problem even after whitelisting the domain . 
here is the error :- 

There seems to be some bug in the code. Please contact support to fix this.
could not validate redirect, please whitelist your domain: https://securedblocks.vercel.app for provided clientId process.env.REACT_APP_CLIENT_ID at https://dashboard.web3auth.io. Also, this project is on cyan network. Please ensure the the used Client ID belongs to this network.

**Reply by Haripandey21** [reply]
hey @himanshuchawla009 need help yr
